lib/auth/methods.go

```diff
@@ -465,16 +465,24 @@ func (s *Server) AuthenticateSSHUser(ctx context.Context, req AuthenticateSSHReq
 		return nil, trace.Wrap(err)
 	}
 
-	// Return the host CA for this cluster only.
-	authority, err := s.GetCertAuthority(ctx, types.CertAuthID{
-		Type:       types.HostCA,
-		DomainName: clusterName.GetClusterName(),
-	}, false)
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
-	hostCertAuthorities := []types.CertAuthority{
-		authority,
+	var hostCertAuthorities []types.CertAuthority
```

Comment:
How difficult would it be to include a test for `AuthenticateSSHUser` that covers both these branches ?